lib/ansible/modules/network/nxos/nxos_config.py

```diff
@@ -249,6 +261,11 @@
     parents: ip access-list test
     before: no ip access-list test
     replace: block
+
+- nxos_config:
```

Comment:
Can you please add a `-name: ` explaining what this is doing (and on the other examples above)

remember to remove the `-` from infront of `- nxos_config`